File: 002/002.cs
using System;

class Program {
    static void Main() {
        int[] numbers = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 };
        int sum = 0;

        foreach (int num in numbers) {
            sum += num;
        }

        Console.WriteLine("Sum: " + sum);
    }
}

File: 002/002.rs
fn main() {
    let numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
    let mut sum = 0;

    for &num in &numbers {
        sum += num;
    }

    println!("Sum: {}", sum);
}